src/OneExplorer/OneExplorer.ts

```diff
@@ -261,8 +261,7 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
         .then(ans => {
           if (ans === approval) {
             Logger.info('OneExplorer', `Delete '${oneNode.node.name}'.`);
-            return vscode.workspace.fs.delete(
-                oneNode.node.uri, {recursive: recursive, useTrash: true});
+            vscode.workspace.fs.delete(oneNode.node.uri, {recursive: recursive, useTrash: true});
           }
```

Comment:
If you want to make the return type consistent, then it's better to return a resolved promise of void type outside the if statement.